examples/systemcalls/systemcalls.c

```diff
@@ -16,7 +25,11 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
-
+    int system_call_return;
+    system_call_return = system(cmd);
+    if(system_call_return == -1){
```

Comment:
@prvbharadwaj1731 does `-1` catch all the possible error condition describe in the comment?